ts/defs.bzl

```diff
@@ -165,11 +166,10 @@ def ts_project(
         buildinfo_out = tsbuildinfo_path if composite or incremental else None,
         emit_declaration_only = emit_declaration_only,
         tsc = tsc,
+        tsc_worker = tsc_worker,
         # TODO: support transpiler attribute when we have a js_library equivalent
         # and can copy over the logic from rules_nodejs to set tsc_js_outs, ts_map_outs
         # and create all the targets for transpilation
         transpile = True,
-        # We don't support this feature at all from rules_nodejs yet
-        supports_workers = False,
```

Comment:
I think we need to add the attribute above with all the others so users can pass True/False?

Reply:
Not really. since we don't have an explicit parameter for it, it'll be passed via kwargs. all good